frontend/packages/ceph-storage-plugin/integration-tests/tests/4-pools/multiple-pool.scenario.ts

```diff
@@ -0,0 +1,100 @@
+import { execSync } from 'child_process';
+import { NS, CLUSTER_STATUS, SECOND } from '../../utils/consts';
+import {
+  createStorageClass,
+  allowExpand,
+  poolDropdown,
+  poolDropdownButton,
+  createPoolDropdown,
+  progressStateCheck,
+  createPool,
+  poolForm,
+  poolStartStateCheck,
+} from '../../views/multiple-pool.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import { browser, element, protractor } from 'protractor';
+
+const cephValue = JSON.parse(execSync(`kubectl get cephCluster -n ${NS} -o json`).toString());
+const cephHealth = cephValue.items[0];
+
+describe('Check availability of Ceph cluster', () => {
+  if (!cephHealth) {
+    it('Should state that Ceph cluster doesnt exist', () => {
+      expect(cephHealth).toBeUndefined();
+    });
+  } else {
+    it('Should state that Ceph cluster exist', () => {
+      expect(cephHealth).toBeDefined();
+    });
+  }
+});
+
+if (cephHealth && cephHealth.status.phase && cephHealth.status.phase !== CLUSTER_STATUS.READY) {
+  describe('Check for pool creation if ceph cluster is in progress', () => {
```

Comment:
Just checking `cephHealth.status.phase !== CLUSTER_STATUS.READY` means that  cephcluster can be warning state as well. So how are you making sure that the cluster is in progress state here in this `it` block?

Reply:
`cephHealth.status.phase` will either report `Failed` 'Created' 'Progressing` or 'Ready`, so we dont need to handle warning state, to avoid confusion, will change the name of variable to `cephStatus`

Reply:
As discussed, please update the text.